README.md

```diff
@@ -85,84 +81,84 @@ Before you get started, please take a few minutes and get acquainted with this a
 
-- While were at it let's go ahead and remove the props from our navigation as well.
+-   While were at it let's go ahead and remove the props from our navigation as well.
 
 **STEP 6 - The final stretch**
 
-- Our cart data is now being provided to us from our `CartContext` time to consume it!
+-   Our cart data is now being provided to us from our `CartContext` time to consume it!
 
-- First, let's head to our `ShoppingCart` component and import the `useContext` hook and our `CartContext`.
+-   First, let's head to our `ShoppingCart` component and import the `useContext` hook and our `CartContext`.
 
-- Now in the component, pass `CartContext` to the `useContext` hook and assign it to a variable named cart.
+-   Now in the component, pass `CartContext` to the `useContext` hook and assign it to a variable named cart.
 
-- Inside of our component we now need to remove all instances of props.
+-   Inside of our component we now need to remove all instances of props.
 
-  - Remove the `props` parameter
-  - Remove the `props` portion in our `getCartTotal` function
-  - Remove `props` when we're mapping over our cart
+    -   Remove the `props` parameter
+    -   Remove the `props` portion in our `getCartTotal` function
+    -   Remove `props` when we're mapping over our cart
 
-- Time to do the same thing for our `Navigation` component.
-  - First import the `useContext` hook and our `CartContext`
-  - Next, pass our `CartContext` to the `useContext` hook and assign it to a variable named cart.
-  - Lastly we need to remove all instances of `props`
-    - Remove `props` from our parameters
-    - Remove `props` from our cart length
+-   Time to do the same thing for our `Navigation` component.
+    -   First import the `useContext` hook and our `CartContext`
+    -   Next, pass our `CartContext` to the `useContext` hook and assign it to a variable named cart.
+    -   Lastly we need to remove all instances of `props`
+        -   Remove `props` from our parameters
+        -   Remove `props` from our cart length
 
 We have now successfully converted our application into using `Context API` 🔥
 
 ## _MVP Requirements:_
 
-- Create a `ProductContext` and a `CartContext`
-- Use the Provider Component from `ProductContext` and `CartContext` to provide data to child components
-- Consume data using the `useContext` hook from `ProductContext` and `CartContext`
+-   Create a `ProductContext` and a `CartContext`
+-   Use the Provider Component from `ProductContext` and `CartContext` to provide data to child components
+-   Consume data using the `useContext` hook from `ProductContext` and `CartContext`
 
 ## Stretch Problems
 
 Do not attempt stretch problems until MVP has been reached and a final commit has been made.
 
-- Create a `removeItem` function that allows you to remove an item from your cart with a click of a button. This `removeItem` function should be able to be consumed from your `ShoppingCartItem` component.
```

Comment:
Try and add this stretch goal it isn't very hard and is something every shopping cart you make will need!